Spec/PushActivationStateMachine.swift

```diff
@@ -634,8 +637,9 @@ class PushActivationStateMachine : QuickSpec {
                         expect(request.httpMethod) == "DELETE"
                         expect(url.query).to(contain(rest.device.id))
                         expect(rest.device.identityTokenDetails).to(beNil())
-                        let authorization = request.allHTTPHeaderFields?["X-Ably-DeviceIdentityToken"]
-                        expect(authorization).to(equal(testIdentityTokenDetails.token.base64Encoded()))
+                        expect(request.allHTTPHeaderFields?["Authorization"]).toNot(beNil())
+                        let deviceAuthorization = request.allHTTPHeaderFields?["X-Ably-DeviceIdentityToken"]
```

Comment:
Still lots of references to this header. These should all be updated to `X-Ably-DeviceToken`. (That's not what the spec says, but that's what's implemented in the system, so we will update those as a separate piece of work.)

Reply:
Done 256c1e5. Thanks